packages/expo-haptics/src/Haptics.ts

```diff
@@ -14,7 +14,7 @@ import { NotificationFeedbackType, ImpactFeedbackStyle } from './Haptics.types';
 export async function notificationAsync(
   type: NotificationFeedbackType = NotificationFeedbackType.Success
 ): Promise<void> {
-  if (!ExpoHaptics.notificationAsync) {
+  if (!ExpoHaptics?.notificationAsync) {
     throw new UnavailabilityError('Haptics', 'notificationAsync');
   }
```

Comment:
I think we eventually want to remove these lines and just re-export `ExpoHaptics` directly from native.